.github/workflows/slack-message.yml

```
@@ -18,7 +18,7 @@ jobs:
       uses: actions/setup-go@v4
 
     - name: Run Test and Send Slack Message
-      run: go run prow-ci/prow_summary.go --no-color
+      run: go run main.go prowjob periodic-slack-report
```

```suggestion
      run: go run main.go periodic-slack-report
```
do we need prowjob before?

---

yep, that command processes prowjob so I thought it makes sense to add it as a subcommand of `prowjob` command

---

makes sense